myhoard/controller.py

```diff
@@ -1408,11 +1434,12 @@ def _purge_old_binlogs(self, *, mysql_maybe_not_running=False):
             self.stats.gauge_float("myhoard.binlog.time_since_any_purged", current_time - last_purge)
             self.stats.gauge_float("myhoard.binlog.time_since_could_have_purged", current_time - last_could_have_purged)
 
-    def _refresh_backups_list(self):
+    def _refresh_backups_list(self, force_refresh: bool = False):
```

Comment:
this requires a lot of changes since `Controller.get_backup_list` is not returning `List[Backup]`, is returning `List[Dict[.....]]`. I rather keep this PR small and not involve any refactoring work that is not required. I'll create an issue for this.

Reply:
Opened https://github.com/Aiven-Open/myhoard/pull/175 for this